shell/platform/glfw/client_wrapper/include/flutter/flutter_window.h

```diff
@@ -14,6 +14,14 @@
 
 namespace flutter {
 
+// A data type for window position and size.
+struct WindowFrame {
+  int left;
```

Comment:
Here and elsewhere, maybe a `size_t` makes more sense.

Reply:
I considered using a more specific type, but the GLFW window size APIs use `int`, so it would have created a mismatch that didn't seem to have an offsetting value given that we have no plan to use this API for a non-GLFW implementation (a GTK or other view-based Linux implementation would use a much different API). If it weren't for the DLL boundary issues on Windows that prevent direct GLFW use, I would just expose the GLFWwindow instead of having wrappers like this.